io/disk/vfc-tests/virtual_fc.py

```diff
@@ -310,44 +326,66 @@ def mpath_verification(self, operation, paths):
+                                   " % (operation, self.path, path_stat))
                     return False
-                return True
-            elif operation == "unmap":
+                else:
+                    self.log.info("opertion:%s path=%s stat=%s return=True \
+                                   " % (operation, self.path, path_stat))
+                    return True
+            elif operation in ('unmap', 'remove'):
                 if path_stat[0] != "failed" or path_stat[2] != "faulty":
+                    self.log.info("opertion:%s path=%s stat=%s return=False \
+                                   " % (operation, self.path, path_stat))
                     return False
-                return True
+                else:
+                    self.log.info("operation:%s path=%s stat=%s return=True \
+                                   " % (operation, self.path, path_stat))
+                    return True
+            else:
+                self.log.info("Operation unknown, provide correct opertion")
 
         for path in paths:
             self.path = path
             if not wait.wait_for(is_path_online, timeout=10):
-                self.err_mesg.append("after %s path %s status did not \
-                                      changed" % (operation, path))
+                self.err_mesg.append("operation:%s path:%s verify failed \
+                                      " % (operation, path))
             else:
-                self.log.info("%s mpath verification success " % operation)
+                self.log.info("oprtion:%s path:%s verify success \
+                               " % (operation, path))
 
-    def test_undefine_define(self):
+    def test_remove_add(self):
         '''
-        Undefin and define the vfchost from vios
+        remove and add back the vfchost from vios
         '''
         self.err_mesg = []
+        self.log.info("\nvfchost remove_add operations\n")
         for _ in range(self.count):
             for vfc_dic in self.dic_list:
-                self.vfchost_define_undefine("undefine", vfc_dic)
+                self.log.info("%s/%s remove operation on \
+                               dic :%s" % (_, self.count, vfc_dic))
+                self.vfchost_add_remove("remove", vfc_dic)
                 time.sleep(self.opp_sleep_time)
-                self.vfchost_define_undefine("define", vfc_dic)
+                self.log.info("%s/%s add operation on \
+                               dic : %s" % (_, vfc_dic, self.count))
+                self.vfchost_add_remove("add", vfc_dic)
                 # sleep time between operations, can be enhanced in future.
                 time.sleep(self.opp_sleep_time)
         if self.err_mesg:
             self.fail("test failed due to folowing reasons:%s" % self.err_mesg)
 
-    def vfchost_define_undefine(self, operation, vfc_dic):
+    def vfchost_add_remove(self, operation, vfc_dic):
```

Comment:
It can be renamed to `vfchost_remove_add`

Reply:
ok